### PR TITLE
Add pylama config to tox.ini (#418)

### PR DESCRIPTION
**Pull request by Nebelhom**
The PR addresses issue #418 

I only added config data to tox.ini as described in the pylama docs ( https://github.com/klen/pylama#id16 )

I could not fully change it, so let's see if the same error persists in appveyor as before.

### CODE REVIEW

**Comment by Nebelhom** [comment]
@whimboo looks like this solved our little problem from #412 

Could you please review?

Thanks

File: tox.ini [hunk 1]
```diff
@@ -1,6 +1,13 @@
 [tox]
 envlist = py27
 
+[pylama:pycodestyle]
+max_line_length = 100
+
+[pylama:pylint]
+max_line_length = 100
+disable = R
+
```
Comment (whimboo) [inline tox.ini]: Lets not put those configurations into the tox.ini bug use setup.cfg for it. See https://pylama.readthedocs.io/en/latest/#configuration-files.

Also I would prefer to have a general pylama section and not specific ones for each tool used. Does `[pylama]` not work? Also what is `disable = R`? Is that really something we need?

**Comment by whimboo** [comment]
Looks like this PR is blocked on issue #420 now.

File: setup.cfg [hunk 2]
```diff
@@ -4,3 +4,9 @@ ignore = C901
 
 [pylama:pep8]
 max_line_length = 100
+
+[pylama:pycodestyle]
+max_line_length = 100
+
+[pylama:pylint]
+max_line_length = 100
```
Comment (whimboo) [inline setup.cfg]: Cannot we define a single pylama section with global settings? I feel it weird that we have to do it for each individual checker.

Comment (Nebelhom) [inline setup.cfg]: I tried that first, but it would cause pycodestyle failures on line_length

With the way I did it, this did not occur...

**Comment by Nebelhom** [comment]
@whimboo Do you have any idea what is going on with all these errors on tests? Everytime, we try to fix one, another one jumps out...

This connection error I am experiencing on my laptop, too. It cannot come from this PR, since I am only changing some lines in setup.cfg and the error is thrown in `scraper.py`...

**Comment by whimboo** [comment]
Johannes please see my latest comment on this PR. This is basically blocked on issue #420 which we have to get fixed first.

File: setup.cfg (hunk 2 above)
Comment (whimboo) [inline setup.cfg]: Ok, looks like it is indeed the way to go. It looks fine to me.

**Comment by whimboo** [comment]
Except OS X which is still running, all other platforms have passing tests now. So I would say lets get this PR merged!

**Pull request merged**